src/features/winterStorage/winterStoragePage/WinterStorageNotice.tsx

```diff
@@ -33,27 +22,14 @@ const WinterStorageNotice = () => {
       <Row>
         <Col lg={{ size: 10, offset: 1 }} xl={{ size: 8, offset: 2 }}>
           <h3>{t('page.winter_storage.notice.title')}</h3>
+          <p>{t('page.winter_storage.notice.paragraph1')}</p>
+          <p>{t('page.winter_storage.notice.paragraph2')}</p>
```

Comment:
This is not a good practise, because in different langauges there can be a different amount of paragraphs. The line breaks should be set in the translations instead and probably a `<pre>`-tag is needed then?

If this is something that works now and will never ever be changed, then this could be acceptable, but instead of using paragraph1 and paragraph2, something like "body text" or something that adds some context should be used instead.